traffic_editor/gui/editor.cpp

```diff
@@ -1837,9 +1837,18 @@ void Editor::mouse_add_edge(
       return;
     }
 
+    const double vertex_radius_meters = 0.1;
+    const double vertex_dist_thresh =
+      project.building.levels[level_idx].drawing_meters_per_pixel /
+      vertex_radius_meters;
```

Comment:
After trying this on a different map, I think this fraction is inverted. It seems like it should really be

```
vertex_radius_meters / project.building.levels[level_idx].drawing_meters_per_pixel;
```